cmd/earthly/org_cmds.go

```diff
@@ -133,9 +77,51 @@ func (app *earthlyApp) orgCmdsPreview() []*cli.Command {
 				},
 			},
 		},
+		{
+			Name:        "member",
+			Aliases:     []string{"members"},
+			Usage:       "Manage organization members",
+			Description: "Manage organization members",
+			UsageText:   "earthly org [--org <organization-name>] members (ls|update|rm)",
+			Subcommands: []*cli.Command{
+				{
+					Name:        "ls",
+					Aliases:     []string{"list"},
+					Usage:       "List organization members and their permission level",
+					Description: "List organization members and their permission level",
+					UsageText:   "earthly org [--org organization] members ls",
+					Action:      app.actionOrgMemberList,
+				},
+				{
+					Name:        "update",
+					Usage:       "Update an organization member's permission",
+					Description: "Update an organization member's permission",
+					UsageText:   "earthly org [--org organization] members update --permission <permission> <user-email>",
+					Action:      app.actionOrgMemberUpdate,
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:        "permission",
+							Usage:       "Update an organization member's permission.",
+							Destination: &app.userPermission,
+						},
+					},
+				},
+				{
+					Name:        "rm",
+					Usage:       "Remove a user from the organization",
+					Description: "Remove a user from the organization",
+					UsageText:   "earthly org [--org organization] members rm <user-email>",
+					Action:      app.actionOrgMemberRemove,
+				},
+			},
+		},
 	}
 }
 
+func (app *earthlyApp) orgCmdsPreview() []*cli.Command {
```

Comment:
Shall I also remove this? I figured we may reuse it at some point in the near future.

Reply:
We can always add it back? No strong opinion.

Reply:
it seems trivial enought to add back in if we need it down the road. I would remove it, but it's not a strong opinion.

Reply:
Cool. I'll remove.

Reply:
You know what - I think it's better to keep the `earthly preview invite` command, but just return an error explaining that this has graduated out of preview. It'll help prevent questions from our users.

Reply:
Sure, that's fine.